package.json

```diff
@@ -1,5 +1,6 @@
 {
   "private": true,
+  "version": "0.18.1",
```

Comment:
Shouldn't it be the version of Studio instead? I don't think we want to version the whole repo.

Reply:
so we want to version only the studio app and ignore the changes that are being made against other packages and apps for now?

Reply:
I'd say yes except for the UI package.

Reply:
@fmvilas So I would know that I got it right this time:
1) we only make a release in GitHub and docker if changes are detected in the `ui` package or `studio` app.

3) the next release version would be `v0.19.0`

2) we track the version in the `studio`'s package.json

Reply:
I think so yeah 😄